crypto/CMakeLists.txt

```diff
@@ -485,6 +485,10 @@ add_library(
 
 target_compile_definitions(crypto_objects PRIVATE BORINGSSL_IMPLEMENTATION)
 
+if(BORINGSSL_PREFIX AND BORINGSSL_PREFIX_SYMBOLS AND GO_EXECUTABLE)
+  add_dependencies(crypto_objects boringssl_prefix_symbols)
+endif()
```

Comment:
Why not place an unconditional dependency on `global_target`?  There might be other problem that arise due to missing transitive dependencies.

```
add_dependencies(crypto_objects global_target)
```

Reply:
That makes sense I think.